### ISSUE TITLE
Crash/freeze a few seconds into Camping Mod game

### ISSUE THREAD

**Issue by ItsCurtis** [Rikmuld/MC-Camping#63]
This mod was working for weeks no problem... Not sure what has changed but it freezes up and crashes shortly after entering the game.

---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 10/8/15 8:05 PM
Description: Tesselating block model

java.lang.NullPointerException: Tesselating block model
    at com.rikmuld.camping.objs.block.Tent.getLightValue(Tent.scala:113)
    at net.minecraft.block.Block.func_176207_c(Block.java:365)
    at net.minecraft.client.renderer.BlockModelRenderer.func_178258_b(BlockModelRenderer.java:112)
    at net.minecraft.client.renderer.BlockModelRenderer.func_178267_a(BlockModelRenderer.java:44)
    at net.minecraft.client.renderer.BlockModelRenderer.func_178259_a(BlockModelRenderer.java:34)
    at net.minecraft.client.renderer.BlockRendererDispatcher.func_175018_a(BlockRendererDispatcher.java:95)
    at net.minecraft.client.renderer.chunk.RenderChunk.func_178581_b(RenderChunk.java:178)
    at net.minecraft.client.renderer.chunk.ChunkRenderWorker.func_178474_a(SourceFile:78)
    at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:38)
    at java.lang.Thread.run(Thread.java:745)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at com.rikmuld.camping.objs.block.Tent.getLightValue(Tent.scala:113)
    at net.minecraft.block.Block.func_176207_c(Block.java:365)
    at net.minecraft.client.renderer.BlockModelRenderer.func_178258_b(BlockModelRenderer.java:112)

-- Block model being tesselated --
Details:
    Block: minecraft:grass[snowy=false]
    Block location: World: (-3031,77,-1494), Chunk: (at 9,4,10 in -190,-94; contains blocks -3040,0,-1504 to -3025,255,-1489), Region: (-6,-3; contains chunks -192,-96 to -161,-65, blocks -3072,0,-1536 to -2561,255,-1025)
    Using AO: false
Stacktrace:
    at net.minecraft.client.renderer.BlockModelRenderer.func_178267_a(BlockModelRenderer.java:44)
    at net.minecraft.client.renderer.BlockModelRenderer.func_178259_a(BlockModelRenderer.java:34)

-- Block being tesselated --
Details:
    Block type: ID #2 (tile.grass // net.minecraft.block.BlockGrass)
    Block data value: 0 / 0x0 / 0b0000
    Block location: World: (-3031,77,-1494), Chunk: (at 9,4,10 in -190,-94; contains blocks -3040,0,-1504 to -3025,255,-1489), Region: (-6,-3; contains chunks -192,-96 to -161,-65, blocks -3072,0,-1536 to -2561,255,-1025)
Stacktrace:
    at net.minecraft.client.renderer.BlockRendererDispatcher.func_175018_a(BlockRendererDispatcher.java:95)

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityPlayerSP['**Its_Curtis**'/273, l='MpServer', x=-2803.70, y=66.42, z=-1417.93]]
    Chunk stats: MultiplayerChunkCache: 2080, 2080
    Level seed: 0
    Level generator: ID 00 - default, ver 1. Features enabled: false
    Level generator options: 
    Level spawn location: -172.00,64.00,256.00 - World: (-172,64,256), Chunk: (at 4,4,0 in -11,16; contains blocks -176,0,256 to -161,255,271), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
    Level time: 809381 game time, 1147422 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
    Forced entities: 114 total; [EntityZombie['Zombie'/261, l='MpServer', x=-2798.51, y=51.00, z=-1455.26], EntityZombie['Zombie'/262, l='MpServer', x=-2798.94, y=51.00, z=-1454.43], EntityWolf['Wolf'/263, l='MpServer', x=-2800.72, y=65.00, z=-1443.00], EntityVillager['Villager'/264, l='MpServer', x=-2814.41, y=67.00, z=-1443.66], EntityVillager['Villager'/265, l='MpServer', x=-2806.94, y=65.50, z=-1444.06], EntityVillager['Villager'/266, l='MpServer', x=-2806.97, y=65.00, z=-1431.31], EntityWolf['Wolf'/267, l='MpServer', x=-2801.53, y=65.00, z=-1424.22], EntityVillager['Villager'/268, l='MpServer', x=-2807.69, y=65.00, z=-1431.31], EntityVillager['Villager'/269, l='MpServer', x=-2807.72, y=65.00, z=-1431.31], EntityVillager['Villager'/270, l='MpServer', x=-2805.88, y=65.00, z=-1439.88], EntityZombie['Zombie'/271, l='MpServer', x=-2798.72, y=51.00, z=-1453.64], EntityWolf['Wolf'/272, l='MpServer', x=-2799.72, y=64.00, z=-1453.31], EntityWolf['Wolf'/274, l='MpServer', x=-2804.41, y=66.00, z=-1421.28], EntityWolf['Wolf'/275, l='MpServer', x=-2800.72, y=65.00, z=-1415.94], EntityWolf['Wolf'/276, l='MpServer', x=-2798.84, y=65.00, z=-1420.50], EntityWolf['Wolf'/277, l='MpServer', x=-2795.94, y=65.00, z=-1416.22], EntityWolf['Wolf'/278, l='MpServer', x=-2799.06, y=65.00, z=-1418.22], EntityWolf['Wolf'/279, l='MpServer', x=-2796.19, y=65.00, z=-1422.38], EntityWolf['Wolf'/280, l='MpServer', x=-2821.56, y=68.00, z=-1426.31], EntityWolf['Wolf'/281, l='MpServer', x=-2820.44, y=68.00, z=-1426.34], EntityBat['Bat'/282, l='MpServer', x=-2789.53, y=46.10, z=-1429.53], EntityZombie['Zombie'/283, l='MpServer', x=-2796.63, y=51.00, z=-1452.50], EntityPlayerSP['**Its_Curtis**'/273, l='MpServer', x=-2803.70, y=66.42, z=-1417.93], EntityBat['Bat'/285, l='MpServer', x=-2791.41, y=46.10, z=-1427.72], EntityXPOrb['Experience Orb'/286, l='MpServer', x=-2789.41, y=63.00, z=-1434.94], EntityXPOrb['Experience Orb'/287, l='MpServer', x=-2792.03, y=64.00, z=-1439.06], EntityItem['item.item.rottenFlesh'/288, l='MpServer', x=-2794.06, y=64.00, z=-1433.34], EntityXPOrb['Experience Orb'/289, l='MpServer', x=-2791.50, y=64.00, z=-1436.59], EntityWolf['Wolf'/290, l='MpServer', x=-2799.09, y=64.00, z=-1429.97], EntityWolf['Wolf'/291, l='MpServer', x=-2795.94, y=64.00, z=-1427.31], EntityWolf['Wolf'/292, l='MpServer', x=-2799.72, y=64.00, z=-1427.31], EntityWolf['Wolf'/293, l='MpServer', x=-2799.44, y=65.00, z=-1424.31], EntitySpider['Spider'/294, l='MpServer', x=-2834.72, y=15.00, z=-1410.72], EntityCreeper['Creeper'/295, l='MpServer', x=-2772.50, y=13.00, z=-1414.50], EntitySquid['Squid'/296, l='MpServer', x=-2774.59, y=57.93, z=-1414.54], EntitySquid['Squid'/297, l='MpServer', x=-2810.75, y=62.46, z=-1386.76], EntitySquid['Squid'/298, l='MpServer', x=-2812.00, y=61.75, z=-1376.94], EntityBat['Bat'/299, l='MpServer', x=-2827.50, y=15.44, z=-1449.25], EntityXPOrb['Experience Orb'/300, l='MpServer', x=-2823.13, y=66.00, z=-1450.16], EntityCreeper['Creeper'/301, l='MpServer', x=-2820.88, y=66.00, z=-1452.34], EntityItem['item.item.rottenFlesh'/302, l='MpServer', x=-2824.94, y=65.00, z=-1451.13], EntityXPOrb['Experience Orb'/303, l='MpServer', x=-2825.03, y=65.00, z=-1451.53], EntityXPOrb['Experience Orb'/304, l='MpServer', x=-2827.22, y=65.00, z=-1451.25], EntitySquid['Squid'/305, l='MpServer', x=-2780.35, y=58.10, z=-1433.97], EntitySquid['Squid'/306, l='MpServer', x=-2772.97, y=59.63, z=-1401.53], EntitySquid['Squid'/307, l='MpServer', x=-2780.19, y=57.85, z=-1400.35], EntitySquid['Squid'/308, l='MpServer', x=-2778.00, y=61.88, z=-1391.72], EntitySquid['Squid'/309, l='MpServer', x=-2787.47, y=49.28, z=-1387.09], EntityBat['Bat'/310, l='MpServer', x=-2852.66, y=43.72, z=-1446.63], EntitySquid['Squid'/311, l='MpServer', x=-2767.22, y=60.38, z=-1454.47], EntitySquid['Squid'/312, l='MpServer', x=-2771.00, y=61.88, z=-1450.09], EntitySquid['Squid'/313, l='MpServer', x=-2777.88, y=59.06, z=-1379.06], EntitySquid['Squid'/314, l='MpServer', x=-2781.00, y=55.09, z=-1381.19], EntityBat['Bat'/315, l='MpServer', x=-2810.25, y=50.10, z=-1462.59], EntitySquid['Squid'/316, l='MpServer', x=-2754.59, y=48.59, z=-1411.53], EntitySquid['Squid'/317, l='MpServer', x=-2754.72, y=59.53, z=-1418.13], EntityCreeper['Creeper'/318, l='MpServer', x=-2812.31, y=13.00, z=-1375.72], EntitySkeleton['Skeleton'/319, l='MpServer', x=-2813.50, y=12.02, z=-1381.75], EntitySpider['Spider'/320, l='MpServer', x=-2850.44, y=70.00, z=-1397.06], EntityBat['Bat'/321, l='MpServer', x=-2857.63, y=39.52, z=-1437.56], EntityHorse['Donkey'/322, l='MpServer', x=-2756.56, y=63.00, z=-1435.94], EntityHorse['Donkey'/323, l='MpServer', x=-2760.41, y=63.00, z=-1434.31], EntitySquid['Squid'/324, l='MpServer', x=-2760.31, y=58.53, z=-1400.34], EntitySkeleton['Skeleton'/325, l='MpServer', x=-2795.25, y=38.00, z=-1369.19], EntitySkeleton['Skeleton'/326, l='MpServer', x=-2792.53, y=37.00, z=-1361.50], EntityZombie['Zombie'/327, l='MpServer', x=-2794.28, y=40.00, z=-1373.22], EntitySquid['Squid'/328, l='MpServer', x=-2785.31, y=52.50, z=-1363.56], EntitySquid['Squid'/329, l='MpServer', x=-2795.19, y=55.84, z=-1372.56], EntityCreeper['Creeper'/330, l='MpServer', x=-2855.78, y=46.00, z=-1380.91], EntityZombie['Zombie'/331, l='MpServer', x=-2862.06, y=36.00, z=-1380.47], EntityZombie['Zombie'/332, l='MpServer', x=-2858.91, y=46.00, z=-1378.53], EntityBat['Bat'/333, l='MpServer', x=-2828.59, y=17.75, z=-1448.59], EntitySquid['Squid'/334, l='MpServer', x=-2756.00, y=62.28, z=-1385.50], EntitySquid['Squid'/335, l='MpServer', x=-2748.53, y=58.72, z=-1387.94], EntitySquid['Squid'/336, l='MpServer', x=-2777.91, y=48.53, z=-1366.03], EntitySquid['Squid'/337, l='MpServer', x=-2781.69, y=52.91, z=-1368.88], EntitySquid['Squid'/338, l='MpServer', x=-2782.13, y=53.97, z=-1363.94], EntityHorse['Horse'/339, l='MpServer', x=-2865.22, y=63.00, z=-1413.84], EntityZombie['Zombie'/340, l='MpServer', x=-2801.34, y=19.91, z=-1358.72], EntityBat['Bat'/341, l='MpServer', x=-2807.53, y=24.81, z=-1360.56], EntityBat['Bat'/342, l='MpServer', x=-2803.53, y=23.10, z=-1357.63], EntityZombie['Zombie'/343, l='MpServer', x=-2868.06, y=68.00, z=-1402.69], EntityBat['Bat'/344, l='MpServer', x=-2873.25, y=47.10, z=-1426.25], EntityHorse['Horse'/345, l='MpServer', x=-2878.75, y=63.00, z=-1436.44], EntityZombie['Zombie'/346, l='MpServer', x=-2866.88, y=63.00, z=-1437.44], EntityHorse['Horse'/347, l='MpServer', x=-2865.31, y=64.00, z=-1435.34], EntityCreeper['Creeper'/348, l='MpServer', x=-2741.56, y=19.00, z=-1430.56], EntityZombie['Zombie'/349, l='MpServer', x=-2739.50, y=19.00, z=-1433.50], EntityZombie['Zombie'/350, l='MpServer', x=-2739.94, y=20.00, z=-1431.53], EntitySkeleton['Skeleton'/351, l='MpServer', x=-2742.72, y=22.00, z=-1437.50], EntityZombie['Zombie'/352, l='MpServer', x=-2744.50, y=19.00, z=-1430.50], EntitySquid['Squid'/353, l='MpServer', x=-2748.16, y=57.53, z=-1409.38], EntityBat['Bat'/354, l='MpServer', x=-2790.31, y=39.08, z=-1350.41], EntitySkeleton['Skeleton'/355, l='MpServer', x=-2787.59, y=36.00, z=-1350.94], EntitySquid['Squid'/356, l='MpServer', x=-2799.34, y=57.00, z=-1359.97], EntityZombie['Zombie'/357, l='MpServer', x=-2860.22, y=67.00, z=-1367.53], EntitySkeleton['Skeleton'/358, l='MpServer', x=-2871.53, y=44.00, z=-1473.97], EntitySkeleton['Skeleton'/359, l='MpServer', x=-2879.28, y=64.00, z=-1443.22], EntityBat['Bat'/360, l='MpServer', x=-2831.22, y=14.10, z=-1484.81], EntityBat['Bat'/361, l='MpServer', x=-2841.53, y=16.10, z=-1476.50], EntitySpider['Spider'/362, l='MpServer', x=-2837.31, y=44.00, z=-1473.91], EntitySquid['Squid'/363, l='MpServer', x=-2750.28, y=60.34, z=-1448.44], EntityZombie['Zombie'/364, l='MpServer', x=-2863.91, y=67.00, z=-1356.41], EntityCreeper['Creeper'/365, l='MpServer', x=-2869.69, y=66.00, z=-1369.53], EntityHorse['Horse'/368, l='MpServer', x=-2875.81, y=68.00, z=-1465.00], EntityZombie['Zombie'/369, l='MpServer', x=-2852.59, y=39.00, z=-1481.06], EntityCreeper['Creeper'/370, l='MpServer', x=-2862.00, y=70.00, z=-1478.41], EntityCreeper['Creeper'/371, l='MpServer', x=-2802.63, y=63.00, z=-1488.06], EntityCreeper['Creeper'/372, l='MpServer', x=-2802.06, y=64.00, z=-1492.06], EntityZombie['Zombie'/373, l='MpServer', x=-2737.16, y=63.00, z=-1470.34], EntityCreeper['Creeper'/375, l='MpServer', x=-2882.84, y=64.00, z=-1444.00], EntityHorse['Horse'/376, l='MpServer', x=-2828.97, y=66.00, z=-1494.03], EntityCreeper['Creeper'/384, l='MpServer', x=-2879.84, y=68.00, z=-1474.25], EntityHorse['Horse'/385, l='MpServer', x=-2881.44, y=67.00, z=-1463.00]]
    Retry entities: 0 total; []
    Server brand: fml,forge
    Server type: Integrated singleplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:351)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2502)
    at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:44)
    at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
    Minecraft Version: 1.8
    Operating System: Mac OS X (x86_64) version 10.11
    Java Version: 1.8.0_60, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 386537168 bytes (368 MB) / 1060372480 bytes (1011 MB) up to 1060372480 bytes (1011 MB)
    JVM Flags: 5 total; -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    IntCache: cache: 1, tcache: 1, allocated: 12, tallocated: 94
    FML: MCP v9.10 FML v8.0.99.99 Minecraft Forge 11.14.3.1450 5 mods loaded, 5 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.05} [Minecraft Coder Pack](minecraft.jar) 
    UCHIJAAAA   FML{8.0.99.99} [Forge Mod Loader](forge-1.8-11.14.3.1450.jar) 
    UCHIJAAAA   Forge{11.14.3.1450} [Minecraft Forge](forge-1.8-11.14.3.1450.jar) 
    UCHIJAAAA   corerm{1.2c} [RikMuld's Core](RikMuldsCore_1.2c.jar) 
    UCHIJAAAA   camping{2.2e} [The Camping Mod 2](TheCampingMod_2.2e.jar) 
    Loaded coremods (and transformers): 
    GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
    Launched Version: 1.8-Forge11.14.3.1450
    LWJGL: 2.9.2
    OpenGL: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

```
Using VBOs: No
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: [1]
Current Language: English (US)
Profiler Position: N/A (disabled)
```

**Comment by ItsCurtis** [comment]
A new world works great with no problems....

**Comment by Rikmuld** [comment]
Now really solved the issue in the update to 1.12.2
